package.json

```diff
@@ -22,11 +23,11 @@
     "node": "14.x"
   },
   "dependencies": {
-    "@octokit/core": "^3.2.1",
-    "@slack/web-api": "^5.14.0",
+    "@octokit/core": "^3.2.4",
```

Comment:
out of curiosity: does it matter if [core](https://www.npmjs.com/package/@octokit/core) or [`@octokit/request`](https://www.npmjs.com/package/@octokit/request) for this project? the size difference doesn't seem relevant really.

Reply:
yea not really as much. just wanted to pick the ones in need but may very well be the full request package (also i love their API design, very HTTP natural/intuitive)